### ISSUE TITLE
Missing VQA-X visual grounding train set?

### ISSUE THREAD

**Issue by CCYChongyanChen** [Seth-Park/MultimodalExplanations#13]
Hi there, in the google drive you share, there is only val & set visual explanation, could you please also share the train set? Many thanks!
![image](https://user-images.githubusercontent.com/21120604/140000218-2142e726-2c1b-40aa-8c69-9ae7bfa21920.png)

**Comment by Seth-Park** [comment]
The visual explanations are only provided for evaluation purposes (i.e. validation set, test set). We did not collect visual explanations for the training set due to high cost. We only provide textual explanations for the train split. Sorry :(
